Assets/_BForBoss/_Weapons/Scripts/BulletSpawner.cs

```diff
@@ -0,0 +1,52 @@
+using Perigon.Utility;
+using Sirenix.OdinInspector;
+using UnityEngine;
+
+namespace Perigon.Weapons
+{
+
+    public enum BulletTypes
+    {
+        RifleNoPhysics,
+        RiflePhysics
```

Comment:
No need to name these specifically Rifle.

Reply:
Do we need an enum for the Bullets type? since it's really only, either Yes Physics, or No Physics.

Reply:
Not necessarily, no - this was more future proofing for rockets/grenades/etc